examples/inference/client.py

```diff
@@ -18,14 +18,36 @@ async def run_main(host: str, port: int, stream: bool = True):
     client = LlamaStackClient(
         base_url=f"http://{host}:{port}",
     )
+    try:
+        models_response = client.models.list()  # Fetch models list from server
+        available_models = [model.identifier for model in models_response]
+        if not available_models:
+            print("No available models in the distribution endpoint.")
```

Comment:
probably do a `cprint(..., "red")`

Reply:
@ashwinb Thank you for your review! pushed the new changes!